samples/downscopedclient.js

```diff
@@ -0,0 +1,111 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+/**
+ * Imports the Google Auth and Google Cloud libraries.
+ */
+const {
+  OAuth2Client,
+  GoogleAuth,
+  DownscopedClient,
+} = require('google-auth-library');
+const {Storage} = require('@google-cloud/storage');
+
+// TODO(developer): Replace these variables before running the sample.
+// Make sure the bucket and object exists in your project.
+// The Cloud Storage bucket name.
+const bucketName = 'your-gcs-bucket-name';
+// The Cloud Storage object name that resides in the specified bucket.
+const objectName = 'your-gcs-object-name';
+
+/**
+ * The following sample demonstrates how to initialize a DownscopedClient using
+ * a credential access boundary and a client obtained via ADC. The
+ * DownscopedClient is used to create downscoped tokens which can be consumed
+ * via the OAuth2Client. A refresh handler is used to obtain new downscoped
+ * tokens seamlessly when they expire. Then the oauth2Client is used to define
+ * a cloud storage object and call GCS APIs to access specified object and
+ * print the contents.
+ */
+async function main() {
+  // Defines a credential access boundary that grants full control over objects
+  // in the specified bucket.
```

Comment:
Looks like this comment is outdated.

Reply:
I don't quite get this, I am just following your [suggestion](https://github.com/googleapis/google-auth-library-nodejs/pull/1311#discussion_r743238383)

Reply:
That comment was when you used the 'storage.objectAdmin' role. Since you changed it, it is no longer accurate.